### DISCUSSION TITLE
After the update, the behavior of the dynamic template has changed

### DISCUSSION THREAD

**Discussion by serggrod** [SilentVoid13/Templater#917] [Help] [answered]
After last updates the behavior of the template has changed  <%+ tp.file.last_modified_date() %>  If earlier it worked only in view mode and it was normal, now it works both when creating a note and when pressing ALT + R and shows the "NaN" value, which is extremely unpleasant since you have to fix this "NaN" every time hands. Can this be fixed somehow?
I have to deliberately distort it in such a way  <+ tp.file.last_modified_date() %>  so that it does not automatically work and does not become NaN, and I add the % sign after creating the note so that the template works correctly - this is very inconvenient

**Comment by Zachatoo** [comment, marked as answer]
I wouldn't expect this to be fixed, as dynamic commands are likely going to be deprecated, see #913.

**Comment by AB1908** [comment]
See also #910
